scripts/qa_crowbarsetup.sh

```diff
@@ -1736,6 +1743,18 @@ function custom_configuration()
                 novanodes="[ ${novanodes%,} ]"
                 proposal_set_value nova default "['deployment']['nova']['elements']['nova-multi-compute-${libvirt_type}']" "$novanodes"
             fi
+
+            if [ -n "$want_sles12" ] && [ -n "$want_docker" ] ; then
+                proposal_set_value nova default "['deployment']['nova']['elements']['nova-multi-compute-docker']" "['$sle12node']"
+                # do not assign another compute role to this node
+                if [ -n "$nodescompute" ] ; then
```

Comment:
Ok, reading it again I think its fine though. Having no compute nodes is a different kind of bug, that should be caught earlier, not here, I agree.